Lombiq.Tests.UI/Extensions/ResponsivenessUITestContextExtensions.cs

```diff
@@ -1,8 +1,7 @@
 using Lombiq.Tests.UI.Constants;
-using Lombiq.Tests.UI.Extensions;
 using Lombiq.Tests.UI.Services;
 
-namespace Lombiq.Tests.UI.Samples.Extensions
+namespace Lombiq.Tests.UI.Extensions
```

Comment:
This was deliberately in the samples. What's a "standard" browser size depends on the project.

Reply:
If so, then we should rename it to something other than "standard".
Standards are decided by an issuer organization (in this case us) as a common baseline. If a project needs to test for phone, ultrawide, etc those are special scenarios and they can make their own custom extension method for those projects.

Reply:
Good point. Let's leave it like this.